client/src/unifycr-wrap.h

```diff
@@ -49,8 +49,8 @@
  * POSIX wrappers: file descriptors
  * --------------------------------------- */
 
-UNIFYCR_WRAP_DECL(open, int, (const char *path, int flags, ...));
-UNIFYCR_WRAP_DECL(write, ssize_t, (int fd, const void *buf, size_t count));
+UNIFYCR_WRAP_DECL(open, int, (const char* path, int flags, ...));
```

Comment:
@adammoody, double check me on this, but you might be able to just delete this file. I don't think we have a `UNIFYCR_WRAP_DECL` macro anymore and these functions are already wrapped in client/src/unifycr-sysio.h. I think this file might be left over from something in the initial days of the project as it hasn't been touched in a year.

Reply:
Ok, I'll take a look.  Thanks for looking at that.  I was just on auto-pilot running through the files here.